### PR TITLE
ouroboros-network:  bump to a fixed version

### PR DESCRIPTION
**Pull request by deepfire**
This bumps consensus to a version that fixes the benchmarking-oriented special case in ledger state setup.

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -1,4 +1,4 @@
-index-state: 2021-12-07T00:00:00Z
+index-state: 2021-12-15T00:00:00Z
```
Comment (nc6) [inline cabal.project:1]: I don't see why this bump is necessary. It's already much newer than that used in consensus.

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs [hunk 2]
```diff
@@ -423,26 +419,31 @@ instance ( ConvertRawHash blk
...
           "Chain selection run for block previously from future: " <> renderRealPointAsPhrase pt
       ChainDB.TraceLedgerReplayEvent ev -> case ev of
         LedgerDB.ReplayFromGenesis _replayTo ->
           "Replaying ledger from genesis"
-        LedgerDB.ReplayFromSnapshot snap tip' _replayTo ->
+        LedgerDB.ReplayFromSnapshot snap tip' (LedgerDB.ReplayStart replayFrom) (LedgerDB.ReplayGoal replayTo) ->
           "Replaying ledger from snapshot " <> showT snap <> " at " <>
-            renderRealPointAsPhrase tip'
-        LedgerDB.ReplayedBlock pt _ledgerEvents replayTo ->
+            renderRealPointAsPhrase tip' <> ".  Pushing"
```
Comment (nc6) [inline cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs:436]:
```suggestion
            renderRealPointAsPhrase tip' <> ".  Replaying"
```

**Review by nc6 — APPROVED** [review]
Looks reasonable to me!

File: cardano-node/src/Cardano/Tracing/Tracers.hs [hunk 3]
```diff
@@ -245,7 +243,8 @@ instance ElidingTracer (WithSeverity (ChainDB.TraceEvent blk)) where
       return (Just ev, count)
   conteliding _tverb _tr ev@(WithSeverity _ (ChainDB.TraceGCEvent _)) (_old, count) =
       return (Just ev, count)
-  conteliding _tverb tr ev@(WithSeverity _ (ChainDB.TraceLedgerReplayEvent (LedgerDB.ReplayedBlock pt [] replayTo))) (_old, count) = do
+                                                                                                                              -- TODO should this be used?
+  conteliding _tverb tr ev@(WithSeverity _ (ChainDB.TraceLedgerReplayEvent (LedgerDB.ReplayedBlock pt [] (LedgerDB.ReplayStart _replayFrom) (LedgerDB.ReplayGoal replayTo)))) (_old, count) = do
```
Comment (newhoggy) [inline cardano-node/src/Cardano/Tracing/Tracers.hs:247]: There is an unused `_replayFrom` here.  Does it need to be used?

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs [hunk 4]
```diff
@@ -490,6 +490,7 @@ instance HasSeverityAnnotation (InboundGovernorTrace addr) where
       InboundGovernor.TrRemoteState {}             -> Debug
       InboundGovernor.TrUnexpectedlyFalseAssertion {}
                                                    -> Error
+      InboundGovernor.TrDemotedToWarmRemote {}     -> Info
```
Comment (newhoggy) [inline cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs:493]: Is this a good log level choice for this event?

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs (hunk 2 above)
Comment (dnadales) [reply cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs:436]: @EncodePanda @Jasagredo do you agree with this suggestion? :point_up:

File: cardano-node/src/Cardano/Tracing/Tracers.hs (hunk 3 above)
Comment (dnadales) [reply cardano-node/src/Cardano/Tracing/Tracers.hs:247]: @Jasagredo @EncodePanda

File: cabal.project (hunk 1 above)
Comment (newhoggy) [reply cabal.project]: I reverted this change.

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs (hunk 2 above)
Comment (EncodePanda) [reply cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs:436]: well, it does not fired during replaying but chain selection and pushing block to the ledger, we have separate events for replaying

**Pull request closed without merging**